tgui/packages/tgui/interfaces/MassSpec.tsx

```diff
@@ -401,7 +401,6 @@ const MassSpectroscopy = (props: SpectroscopyProps) => {
       {/* Sliders */}
       <Slider
         step={graphUpperRange / base_width}
-        suppressFlicker
```

Comment:
fun fact, this was the only place where `suppressFlicker` was used. it takes a number! the reason why this didn't cause an error was because it just doesn't work at all under the hood, either
https://github.com/tgstation/tgstation/blob/master/tgui/docs/component-reference.md#slider